src/pass.h

```diff
@@ -87,6 +88,9 @@ struct PassRunner {
   void setValidateGlobally(bool validate) {
     options.validateGlobally = validate;
   }
+  void setFeatures(FeatureSet features) {
```

Comment:
we don't strictly need this (can use the constructor as mentioned above, or also do `getPassOptions().features = `) but I suppose this could be a useful convenience

Reply:
I figured we'd want to be able to set this without optLevel (see above)